WordPress/src/main/java/org/wordpress/android/ui/posts/PostSettingsUtils.kt

```diff
@@ -40,7 +40,7 @@ class PostSettingsUtils
             } else if (postUtilsWrapper.isPublishDateInTheFuture(postModel.dateCreated)) {
                 labelToUse = resourceProvider.getString(R.string.schedule_for, formattedDate)
             } else {
-                labelToUse = resourceProvider.getString(R.string.publish_on, formattedDate)
+                labelToUse = resourceProvider.getString(R.string.immediately)
```

Comment:
What would you think about adding a test for this change in `PostSettingsUtilsTest`?